source/_patterns/03-templates/single-column-grid/single-column-grid.twig

```diff
@@ -1,3 +1,5 @@
 <div class="grid grid--single-column">
 
+    {% include 'atoms-text' with {nodes: content}  %}
```

Comment:
This isn't working. Think I'm missing a loop, all experiments so far haven't done what I'd expected.